internal/util/setters/setters.go

```diff
@@ -423,7 +427,8 @@ func parentDirWithKptfile(parentPath string) (string, error) {
 
 // DefExists returns true if the setterName exists in Kptfile definitions
 func DefExists(resourcePath, setterName string) bool {
-	if err := openapi.AddSchemaFromFile(filepath.Join(resourcePath, kptfile.KptFileName)); err != nil {
+	_, err := openapi.AddSchemaFromFile(filepath.Join(resourcePath, kptfile.KptFileName))
```

Comment:
It looks like we are adding a schema here, but never clean it up?

Reply:
Yes. I just replaced the existing logic. Since it is not deleted, I din't clean it up. This logic is correct for now. Removing schema as part of this method will lead to undesirable results. I will refactor it in the major changes PR.